specification/servicebus/resource-manager/readme.md

```diff
@@ -67,6 +67,25 @@ input-file:
 - Microsoft.ServiceBus/preview/2021-06-01-preview/CheckNameAvailability.json
 ```
 
+### Tag: package-2022-01-preview
+
+These settings apply only when `--tag=package-2022-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2022-01-preview'
```

Comment:
Do you need to change the default tag https://github.com/Azure/azure-rest-api-specs/pull/17967/files#diff-ead548efa4d6b19188b90bf9ea7b28c58b216031d7f66d89782588e4e534fbd9R29 as  package-2022-01-preview ?

Reply:
Yes. Thanks for noticing that. Have fixed that and pushed the commit